src/Service/AuthenticationService.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Markette\GopayInline\Service;
+
+use Markette\GopayInline\Api\Lists\Scope;
+use Markette\GopayInline\Exception\HttpException;
+
+class AuthenticationService extends AbstractService
+{
+
+	/**
+	 * @param string $scope
+	 * @return bool
+	 */
+	public function verifyCredentials($scope = Scope::PAYMENT_ALL)
```

Comment:
If there won't be another verification methods then it would be def better. What do you think?